Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM rust as builder
+WORKDIR /src/bitcoin-p2p-firewall
+COPY . .
+RUN cargo install --target x86_64-unknown-linux-gnu --path /src/bitcoin-p2p-firewall
```

Comment:
Installing glibc should not be necessary.  It works just fine on musl.

Reply:
thanks! this is what i was looking for. will update

Reply:
updated in new commit